wherehows-web/app/typings/api/datasets/relationships.d.ts

```diff
@@ -23,7 +23,8 @@ interface IDatasetLineage {
   dataset: IDatasetView;
   type: string;
   actor: string;
-  modified: string | undefined;
 }
 
-export { Relationships, RelationshipType, IDatasetLineage };
+type LineageList = Array<IDatasetLineage>;
+
+export { Relationships, RelationshipType, IDatasetLineage, LineageList };
```

Comment:
Is the `Relationships` type still used?

Reply:
Guess I'll just delete it and find out lol